### ISSUE TITLE
Does wine-tkg-git automatically apply more patches over wine-staging?

### ISSUE THREAD

**Issue by Joe23232** [Frogging-Family/wine-tkg-git#251]
Does `wine-tkg-git` automatically apply more patches over `wine-staging`?

**Comment by Tk-Glitch** [comment]
Yes. The list is available here for the default config: https://github.com/Tk-Glitch/wine-tkg/blob/master/wine-tkg-config.txt

For a different config or proton-tkg, the default patches list will be different of course (you can find the default for proton-tkg here: https://github.com/Tk-Glitch/wine-proton-tkg/blob/master/wine-tkg-config.txt).
